scripts/module/lint.sh

```diff
@@ -30,4 +39,4 @@ done
 
 echo "Lint module ${TARGET}"
 eval "$TFLINT_CMD"
-rm tmp-outputs.tf
+rm ${out_file}
```

Comment:
`rm -f` to avoid error in case of no common-variables ?

Reply:
good catch

Reply:
should be better now